src/main.cc

```diff
@@ -24,20 +25,26 @@ int main(int argc, char **argv) try {
   osmium::io::Reader reader{infile};
   osmium::io::Writer writer{outfile};
 
-  ExitToRewriter xform;
+  DestinationsRewriter destinationsRewriter;
+  ExitsRewriter exitsRewriter;
 
   while (const auto inbuf = reader.read()) {
-    osmium::apply(inbuf, xform);
+    // Todo: refactor as n-stages pipeline with passes enabled/disabled from cmdline
```

Comment:
no there is no ticket for this; it's just something we probably want once we implement more rewriters (?)

more of an idea for the future